src/redux/book/book.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'redux/books/REMOVE_BOOK';
+const initialState = {
+  Books: [],
+};
+export const Addbooks = (books) => ({
+  type: ADD_BOOK,
+  text: books,
+});
+export const removeBooks = (books) => ({
+  type: REMOVE_BOOK,
+  text: books,
+});
+export default function bookReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return {
+        state,
+        Books: [state.Books, action.text],
+      };
+    case REMOVE_BOOK:
+      return {
+        state,
```

Comment:
- [ ] I kindly suggest that you please make use ES6 filter() method to create a new array, which will not contain the book you want to remove from the store (filter by the id key - i.e.:
return state.filter(book => book.id !== id)